site/src/pages/components/menu.explainer.mdx

```diff
@@ -81,22 +81,25 @@ model. See https://github.com/openui/open-ui/issues/1194 for more discussion abo
 
 `<menubar>`
 
-* Menubar should be used to represent a series of `<menuitem>`s that can then open submenus of commands.
-* In a menubar, a menuitem’s most common purpose is to invoke submenus (menulists).
-* An **in-page** menu that can contain a list of `<menuitem>`
-* Menuitems within a menubar are displayed horizontally by default.
-* Supports :disabled, disabled attribute
+* Menubars commonly represent a series of `<menuitem>`s that are used to open their own submenus (menulists) of commands.
+* Unlike menulists, menubars are rendered **in-page** rather than as a popover, so their menuitems are persistently available.
```

Comment:
we could consider saying “part of normal flow” here instead of “in-page”? (like [in CSS](https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/CSS_layout/Introduction#normal_layout_flow))

Reply:
that's fine by me.  i just kept that term as it was previously written

Reply:
```suggestion
* Unlike menulists, menubars are rendered part of normal flow rather than as a popover, so their menuitems are persistently available.
```